cmd/vic-machine/common/ops_credentials.go

```diff
@@ -47,6 +48,12 @@ func (o *OpsCredentials) Flags(hidden bool) []cli.Flag {
 			Usage:  "Password or token for the operations user. Defaults to the credential supplied with target",
 			Hidden: hidden,
 		},
+		cli.BoolFlag{
+			Name:        "ops-init-perms",
```

Comment:
Does `vic-machine configure` apply to this option? Ping @lcastellano @andrewtchin @mdubya66 - we made the call that for any new `vic-machine` option, `configure` support should be implemented simultaneously if applicable.

Reply:
Working with George on vic-machine-configure..